build-tools/Dockerfile.rhel7.runtime

```diff
@@ -0,0 +1,51 @@
+FROM registry.access.redhat.com/rhel-atomic
+
+LABEL name="f5networks/k8s-bigip-ctlr" \
+      vendor="F5 Networks" \
+      version="1.2" \
+      release="1" \
```

Comment:
The version should be passed in with the docker build `--label` option, so that the version can be recorded in one place.